README.markdown

```diff
@@ -78,4 +77,4 @@ We Are Working On
 * The aggregation functionality is ignored completely
 * The code is... a mess. Deadlines do that to you, we're working on it
 * The default table schema isn't indexed all the places it needs to be
-* Easier ways to diff URLs
\ No newline at end of file
+* Easier ways to diff URLs
```

Comment:
Same thing happened again! I didn't touch this line and it keeps picking something up here. I'm editing via github directly so it's not my IDE settings or anything. There's no change here so I guess we'll just have to leave it like this?

Reply:
Sure.